ConceirgeDinning.Adapter.USRestaraunt/Models/Geo.cs

```diff
@@ -0,0 +1,8 @@
+﻿namespace ConceirgeDinning.Adapter.USRestaraunt.Models
+{
+    public class Geo
```

Comment:
Naming should be proper.
No short form

Reply:
It is according to the contract provided by the supplier